db/migrate/20240620024938_add_back_field_type_enum_to_user_fields.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class AddBackFieldTypeEnumToUserFields < ActiveRecord::Migration[7.0]
+  def change
+    if !ActiveRecord::Base.connection.column_exists?(:user_fields, :field_type_enum)
```

Comment:
Only runs if the (now-neutered) post-migration that renames `field_type_enum` has already been run.